src/cargo/ops/cargo_test.rs

```diff
@@ -429,4 +430,13 @@ fn report_test_error(
     }
 
     crate::display_error(&err, &mut ws.config().shell());
+
+    let harness: bool = unit_err.unit.target.harness();
+    let nocapture: bool = test_args.contains(&"--nocapture");
+
+    if !is_simple && harness && !nocapture {
+        drop(ws.config().shell().note(
+            "test was terminated via a signal. stderr might be truncated. use `--nocapture` flag to disable output buffering."
```

Comment:
I feel like this is long enough that its going to lose users (and technically a test can exist from something besides a signal, like `std::process::exit`)

How about

> "test exited abnormally; to see the full output pass `--nocapture` to the harness"